documentation/operators/src/troubleshooting/vps-isp.md

```diff
@@ -23,8 +23,48 @@ Begin with the steps listed in [*Connectivity Test and Configuration*](../nodes/
 4. Search or ask your ISP for additional documentation related to IPv6 routing and ask them to provide you with `IPv6 IP address` and `IPv6 IP gateway address`
 - For example Digital Ocean setup isn't the most straight forward, but it's [well documented](https://docs.digitalocean.com/products/networking/ipv6/how-to/enable/#on-existing-droplets) and it works.
 
-5. Search for guides regarding your particular system and distribution. For Debian based distributions using systemd, some generic guides such as [this one](https://cloudzy.com/blog/configure-ipv6-on-ubuntu/) work as well.
+5. Search for guides regarding your particular system and distribution. For Debian based distributions using systemd, some generic guides such as [this one](https://cloudzy.com/blog/configure-ipv6-on-ubuntu/) or [this one](https://help.ovhcloud.com/csm/en-ie-vps-configuring-ipv6?id=kb_article_view&sysparm_article=KB0047567) work as well.
 
+### Network configuration
+
+On modern Debian based Linux distributions, network is being configure by either [Netplan](www.netplan.io) or [ifup/ifdown] utilities (https://manpages.debian.org/testing/ifupdown/ifup.8.en.html). It is very easy to check which one you have.
+
+1. If you have the following folder `/etc/netplan` which has got a YAML file - you are likely to have Netplan.
+2. If you have the following folder `/etc/network` and it is not empty - you are likely to have ifup/down.
+
+Most contemporary Ubuntu/Debian distributions come with Netplan, however it is possibly that your hosting provider is using a custom version of ISO. For example, Debian 12 (latest version as of June 2024) may come with ifup/down.
+
+Nym operator community members have tested a VPS with Netplan and where in some cases `nym-node --mode exit-gateway` was not routing IP packets properly even after running `network_tunnel_manager.sh` script. We are working on a guide to setup Netplan configuration manually for such cases.
+
+Configuration of ifup/ifdown is a bit simpler. If the `network_tunnel_manager.sh` script doesn't do the job, open `/etc/network/interfaces` file (research if your system uses a different naming for it) and configure it similarly to this:
+
+```sh
+auto lo
+iface lo inet loopback
+
+auto eth0
+iface eth0 inet static
+address <YOUR_IPV4_ADDRESS>
+netmask NETMASK
+
+gateway <YOUR_IPV4_GATEWAY>
+iface eth0 inet6 static
+        accept_ra 0
+        address <YOUR_IPV6_ADDRESS>
+        netmask 64
+        gateway <YOUR_IPV6_GATEWAY>
+post-up /sbin/ip -r route add <YOUR_IPV6_GATEWAY> dev eth0
+post-up /sbin/ip -r route add default via <YOUR_IPV6_GATEWAY>
+```
+Last two lines are particularly important as they enable IPv6 routing. 
```

Comment:
I would add a note how an operator can find their IPv6 Gateway as they may be all confused with Nym node gateway :/